### PR TITLE
Create finetuningresearch python package, allowing it to be pip insta…

### PR DESCRIPTION
**Pull request by shankarg87**
…lled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,21 @@ Use a tag versioning by date / user as needed. For example,
 docker build . -t rparundekar/fine_tune_research:20230110_01
 docker push rparundekar/fine_tune_research:20230110_01
 ```
+
+## Library
+To use this finetuning library as a python package, perform a pip install directly from github. This should install all dependencies as well.
+
+```sh
+pip install -v git+https://github.com/shankarg87/training_research@main
```
Comment (rparundekar) [inline README.md:27]: For this repo, should this be something different?

**Review by rparundekar — COMMENTED** [review]
@shankarg87 - why move the code to src/finetuningresearch (personally. I prefer app or just src if you don't like app)

Also, the imports with .sft and .utils should not be there (it's sort of bad) - maybe update PYTHONPATH instead? Works inside the container since app is on the path.

**Comment by rparundekar** [comment]
Great stuff! Do you think we should just add a Notebook so people can try the instructions for launch directly?

**Comment by shankarg87** [comment]
> Great stuff! Do you think we should just add a Notebook so people can try the instructions for launch directly?

will submit the PR into the orchestration repo

File: README.md (hunk 1 above)
Comment (shankarg87) [reply README.md]: fixed, ptal

**Review by rparundekar — APPROVED** [review]

**Pull request merged**